src/sdl/Database.cpp

```diff
@@ -0,0 +1,128 @@
+#include "Database.h"
+
+#include "../util/Constants.h"
+
+#include <iostream>
+#include <filesystem>
+#include <fstream>
+#include <nlohmann/json.hpp>
+#include <nlohmann/json_fwd.hpp>
+#include <vector>
+
+Database::Database() {
```

Comment:
I forgot to make sure the database submodule was initialised lol. Please add a little log to say whether the database was successfully found or not